src/main/frontend/handler/editor.cljs

```diff
@@ -1286,6 +1287,7 @@
       (let [page-name (string/lower-case page)]
         (state/clear-edit!)
         (insert-first-page-block-if-not-exists! page-name)
+        (recent-handler/add-page-to-recent! (state/get-current-repo) page-name)
```

Comment:
Hi, how do you think we move `add-page-to-recent!` into `redirect-to-page!`, so that we don't need to add it in other places?

Reply:
This sounds like a good idea to me. I'm not yet aware of the big picture of logseq so I wanted to do a minimal-impact change first.

I'll add `add-page-to-recent!` to `redirect-handler!` and see if it works out :-) Thanks!